lib/rack/headers.rb

```diff
@@ -4,6 +4,22 @@ module Rack
   # (by using non-lowercase response header keys), automatically handling
   # the downcasing of keys.
   class Headers < Hash
+    KNOWN_HEADERS = {}
+    %w(
+      Accept-CH Access-Control-Allow-Origin Access-Control-Allow-Credentials Access-Control-Expose-Headers Access-Control-Max-Age Access-Control-Allow-Methods Access-Control-Allow-Headers
```

Comment:
Could we please sort these alphabetically and have one header per line?

Reply:
Sure! I did so.